content/actions/hosting-your-own-runners/monitoring-and-troubleshooting-self-hosted-runners.md

```diff
@@ -57,10 +57,21 @@ $ cat ~/actions-runner/.service
 actions.runner.octo-org-octo-repo.runner01.service
 ```
 
+If that fails due to the service being installed elsewhere, the service name can be obtained from the list of running services. For example:
+
+```shell
+$ systemctl --type=service | grep actions.runner
+  actions.runner.org-repo.hostname.service loaded active running GitHub Actions Runner (org-repo.hostname)
```

Comment:
For consistency, I think it would be better to stick to `octo-org` and `octo-repo` as the example organization and repository names throughout:

```suggestion
actions.runner.octo-org-octo-repo.hostname.service loaded active running GitHub Actions Runner (octo-org-octo-repo.hostname)
```